modules/invisiblyAnalyticsAdapter.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * invisiblyAdapterAdapter.js - analytics adapter for Invisibly
+ */
+import { ajaxBuilder } from '../src/ajax';
+import adapter from '../src/AnalyticsAdapter';
+import adapterManager from '../src/adapterManager';
+
+const DEFAULT_EVENT_URL = 'https://api.pymx5.com/v1/' + 'sites/events';
+const analyticsType = 'endpoint';
+const analyticsName = 'Invisibly Analytics Adapter:';
+
+const utils = require('../src/utils');
+const CONSTANTS = require('../src/constants.json');
+const ajax = ajaxBuilder(0);
+
+// Events needed
+const {
+  EVENTS: {
+    AUCTION_INIT,
+    AUCTION_END,
+    BID_ADJUSTMENT,
+    BID_TIMEOUT,
+    BID_REQUESTED,
+    BID_RESPONSE,
+    NO_BID,
+    BID_WON,
+    BIDDER_DONE,
+    SET_TARGETING,
+    REQUEST_BIDS,
+    ADD_AD_UNITS,
+    AD_RENDER_FAILED
+  }
+} = CONSTANTS;
+
+const _VERSION = 1;
+const _pageViewId = utils.generateUUID();
+let initOptions = null;
+let _startAuction = 0;
+let _bidRequestTimeout = 0;
+let flushInterval;
+let invisiblyAnalyticsEnabled = false;
+
+const w = window;
+const d = document;
+let e = d.documentElement;
+let g = d.getElementsByTagName('body')[0];
+let x = w.innerWidth || e.clientWidth || g.clientWidth;
+let y = w.innerHeight || e.clientHeight || g.clientHeight;
+
+let _pageView = {
+  eventType: 'pageView',
+  userAgent: window.navigator.userAgent,
+  timestamp: Date.now(),
+  timezoneOffset: new Date().getTimezoneOffset(),
+  language: window.navigator.language,
+  vendor: window.navigator.vendor,
+  screenWidth: x,
+  screenHeight: y
+};
+
+let _eventQueue = [_pageView];
+
+let invisiblyAdapter = Object.assign(
+  adapter({ url: DEFAULT_EVENT_URL, analyticsType }),
+  {
+    track({ eventType, args }) {
+      handleEvent(eventType, args);
+    },
+    sendEvent
+  }
+);
+
+invisiblyAdapter.originEnableAnalytics = invisiblyAdapter.enableAnalytics;
+invisiblyAdapter.enableAnalytics = function(config) {
+  initOptions = config.options || {};
+  initOptions.url = initOptions.url || DEFAULT_EVENT_URL;
+  if (initOptions.url && initOptions.account) {
+    invisiblyAnalyticsEnabled = true;
+    invisiblyAdapter.originEnableAnalytics(config);
+  } else {
+    invisiblyAnalyticsEnabled = false;
+    invisiblyAdapter.originDisableAnalytics();
+  }
+  flushInterval = setInterval(flush, 1000);
```

Comment:
Given a normal auction workflow - this `interval` is still active after the auction finishes.  

I see the variable assigned here `flushInterval` gets cleared when the `disableAnalytics` is called but not anywhere else.  Would it make sense to call the `clearInterval()` again in the `auctionEnd` part of the `sendEvent` function (to clean-up after the last flush)?

If you need this interval running after the auction has finished, can you please clarify on the use-case?

Reply:
Agreed @jsnellbaker, have pushed changes to invoke clearInterval() again in the auctionEnd part of the sendEvent function to clean-up after the last flush.

Let me know if any additional changes are needed.
Thanks.

Reply:
@jsnellbaker,
Had added the `clearInterval()` as you suggested, but for some reason, the CircleCI job failed (could you provide some insights?) Have reverted the change & both the CI jobs passed. Is this a blocking change? The interval would be flushed anyways when adapter is disabled. 
Will re-commit it if you can give a go-ahead with failing CircleCi job or suggest changes to prevent job from failing.
Regards,
Pankaj

Reply:
The circleCI tests haven't been entirely stable as of late (unfortunately), so it's possible that the reason it failed wasn't due to your change.

Can you please reintroduce the change to the file so I can take a look?

Reply:
Ok @jsnellbaker. Have reintroduced the change to invoke `clearInterval()` from `sendEvent()` when `AUCTION_END` event is triggered.
Let me know if any additional changes are needed.